health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CreateCampaign.js

```
@@ -292,6 +315,50 @@ const CreateCampaign = ({ hierarchyType, hierarchyData }) => {
+              variation={"primary"}
+              label={t("ES_CAMPAIGN_DELIVERY_SUBMIT")}
+              onClick={() => {
+                setShowPopUp(false);
+                if (pendingFormData) {
+                  handleCampaignMutation(pendingFormData, true);
+                }
+              }}
+            />,
+          ]}
+          sortFooterChildren={true}
+        ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)